### ISSUE TITLE
Song fading and triggers break when a song reverse-fades

### ISSUE THREAD

**Issue by zaeonNineZero** [TheComputerizer/Music-Triggers#18]
**TL;DR:**
Songs reverse-fading breaks trigger processing and makes the music ignore the volume slider.

**Details:**
If a song reverse-fades because its trigger(s) got reactivated, the song will completely ignore the music volume slider (similar to #15 ) and not fade out when its trigger is no longer active.  This also breaks triggers completely, as the song will continue to play even when it should fade out and stop.

The issue semi-fixes itself when the affected song finishes, as the next song loads at the proper volume, but the issue instantly returns when the song tries to fade out and stop.  This also occurs if the music volume slider is set to 'off' and then back any non-zero value.

Exiting the world and reloading it fixes the issue until a song tries to reverse-fade, at which point the issue returns.

**Comment by Huetete** [comment]
This happens to me as well in 1.16

**Comment by TheComputerizer** [comment]
Noted and will be fixed this weekend

**Comment by TheComputerizer** [comment]
This has been fixed in 5.2

**Comment by roccscout** [comment]
https://streamable.com/1alaq3
